src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledNoticeGroupDefinition.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.circulation.domain.notice.schedule;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.folio.circulation.domain.notice.NoticeFormat;
+import org.folio.circulation.domain.notice.NoticeTiming;
+
+public class ScheduledNoticeGroupDefinition {
+
+  public static ScheduledNoticeGroupDefinition from(ScheduledNotice notice) {
+    ScheduledNoticeConfig config = notice.getConfiguration();
+
+    return new ScheduledNoticeGroupDefinition(notice.getRecipientUserId(), config.getTemplateId(),
+      notice.getTriggeringEvent(), config.getFormat(), config.getTiming());
+  }
+
+  private final String userId;
+  private final String templateId;
+  private final TriggeringEvent triggeringEvent;
+  private final NoticeFormat noticeFormat;
+  private final NoticeTiming noticeTiming;
+
+
+  public ScheduledNoticeGroupDefinition(String userId, String templateId,
+                                        TriggeringEvent triggeringEvent,
+                                        NoticeFormat noticeFormat,
+                                        NoticeTiming noticeTiming) {
+    this.userId = userId;
+    this.templateId = templateId;
+    this.triggeringEvent = triggeringEvent;
+    this.noticeFormat = noticeFormat;
+    this.noticeTiming = noticeTiming;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+
+    if (o == null || getClass() != o.getClass()) return false;
+
```

Comment:
Hi @dmytrotur, could you please fix checkstyle here:
 ```suggestion
if (this == o) {
 return true;
}
if (o == null || getClass() != o.getClass()) { 
 return false;
}
```

Reply:
@bohdan-suprun Done.